packages/frontend/editor-ui/src/components/ResourceMapper/ResourceMapper.vue

```diff
@@ -436,8 +438,8 @@ function fieldValueChanged(updateInfo: IUpdateInformation): void {
 	let newValue = null;
 	if (
 		updateInfo.value !== undefined &&
-		updateInfo.value !== '' &&
 		updateInfo.value !== null &&
+		(props.isAllowEmptyValues || updateInfo.value !== '') &&
```

Comment:
@MiloradFilipovic , you had thoughts on possibly allowing other empty values too - I now only allowed the bypass for `''` here, but would you rather have this logic be something like

`props.isAllowEmptyValues || (updateInfo.value !== undefined && updateInfo.value !== null && updateInfo.value !== ''`

Reply:
I was thinking about solving the issue on the front-end but now thinking about it this is more sensible than removing this condition that is there from the initial implementation.